lib/source/consul.js

```diff
@@ -24,9 +24,9 @@ class Consul extends Source.Polling(Parser) { // eslint-disable-line new-cap
    */
   constructor(name, opts) {
     const options = Object.assign({
-      host: Consul.DEFAULT_ADDRESS,
-      port: Consul.DEFAULT_PORT,
-      secure: false
+      host: Config.get('consul:host'),
+      port: Config.get('consul:port'),
+      secure: Config.get('consul:secure')
     }, opts);
```

Comment:
Do you think it's worth doing this?
```javascript
const options = Object.assign({
  host: Consul.DEFAULT_ADDRESS,
  port: Consul.DEFAULT_PORT,
  secure: false
}, Config.get('consul'), opts);
```
That way we get defaults in case users leave something out of the config. Plus, we still get to overwrite options in the constructor for testing.